README.md

```diff
@@ -313,24 +313,28 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   - [WITI](http://www.witi.com/)
 
     WITI is the premier global organization helping tech-savvy women attain their professional goals. WITI organizes events and summits for women.
-    
+
   - [Women in Tech Summit](http://www.innotechconferences.com/dallas/about/women-in-tech-summit)
 
-  The Women in Tech Summit is a special event designed specifically for women in IT. This event includes a fireside chat, two panel presentations, lunch and networking.
+    The Women in Tech Summit is a special event designed specifically for women in IT. This event includes a fireside chat, two panel presentations, lunch and networking.
```

Comment:
<img src="http://s2.quickmeme.com/img/f1/f11593a1ef9d3bab8d5421b4aab400a78a3a67e1a3c185791cff5f39ba936524.jpg" />
I had noticed it a while ago and didn't had time to fix